docs/en/fine_tuning/how_to/creatings.mdx

```diff
@@ -0,0 +1,125 @@
+---
+weight: 30
+---
+
+# Creating Fine-Tuning Tasks
+
+## Preparation
+
+### Datasets
+
+We support reading datasets from your S3 storage as well as from internal datasets.
+If you want to use S3 storage, note that you need to add the following annotations to the corresponding Secret `s3-url`, `s3-name`, `s3-path`.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-credentials
+  namespace: fy-c1 #[!code callout]
+  annotations:
+    s3-url: http://minio-service.kubeflow.svc.cluster.local:9000/finetune #[!code callout]
+    s3-name: test-minio #[!code callout]
+    s3-path: coco128 #[!code callout]
+  labels:
+    aml.cpaas.io/part-of: aml
+type: Opaque
+data:
+  AWS_ACCESS_KEY_ID: bWluaW8= #[!code callout]
+  AWS_SECRET_ACCESS_KEY: bWluaW8xMjM= #[!code callout]
+```
+<Callouts>
+  1. Replace with your own namespace.
+  2. This item is the endpoint/bucket.
```

Comment:
Set to your S3 storage service endpoint and bucket like `https://endpoint:port/bucket`